index.html

```diff
@@ -14,19 +14,20 @@
     />
   </head>
   <body>
-    <header>
+    <header id="top-header">
       <nav id="mobile-navbar">
-        <img src="assets/menu.png" alt="" />
+        <img src="assets/menu.png" alt="" id="openbtn"/>
       </nav>
     </header>
     <section id="headline-section">
       <header id="navbar-grid">
+        <a href="#" id="closebtn">&times;</a>
         <nav id="desktop-navbar">
-            <a href="#home">Portfolio</a>
-            <a href="#About">About</a>
-            <a href="#Contact">Contact</a> 
-    </nav>
-    </header>
+            <a href="#portolioworks" id="menu-link1">Portfolio</a>
+            <a href="#About" id="menu-link2">About</a>
+            <a href="#Contact" id="menu-link3">Contact</a> 
```

Comment:
- [ ] Kindly modify the `href=` to navigate to the correct part of the portfolio when an option in the mobile menu it's clicked

Reply:
sure I am just updating it